### ISSUE TITLE
startLicensingSilently doesn't mark productActivated as true

### ISSUE THREAD

**Issue by charlieMonroe** [PaddleHQ/Mac-Framework#105]
Generally, I want to know if the license is activated without showing the UI - so that I can change the UI of the main window, etc. If I call `startLicensing(timeTrial:with:)`, I immediately get Paddle UI.

So I've tried switching to `startLicensingSilently` - which correctly loads the `activatedEmail()` and `activatedLicenceCode()`, but `productActivated()` still returns `false` - which is not correct.

When the non-silent version is called, `productActivated()` returns true correctly.

Here's some code:

```
paddle.startLicensingSilently(infoDict, timeTrial: false)
paddle.productActivated() // Returns false
paddle.activatedEmail() // test@charliemonroe.net
paddle.activatedLicenceCode() // B289291C-9BC8B360-XXXXX-6473E725-160D44CE
```

**Comment by avalore** [comment]
Thanks, this looks to be related to non-time-trials and silent licencing. This has been resolved and will be available in the next release (v3.1.0)

**Comment by charlieMonroe** [comment]
Thanks!
